src/qt/sendcoinsdialog.h

```diff
@@ -60,6 +60,7 @@ public Q_SLOTS:
     Ui::SendCoinsDialog *ui;
     ClientModel *clientModel;
     WalletModel *model;
+    std::unique_ptr<WalletModelTransaction> m_current_transaction;
```

Comment:
This looks like an ugly hack? Why is it not just passed as a reference to `PrepareSendText`?

Reply:
`on_sendButton_clicked()` also needs access to `m_current_transaction`

Reply:
This never reset, maybe clear it after send?